src/main.ts

```diff
@@ -80,6 +115,12 @@ export class Minifier {
     }
   }
 
+  private renameIdent(node: ts.Node) {
+    return this.renameProperty(node.getText());
```

Comment:
should this call this.ident(node) inside, in case you change that at some point?